jeremiahmegel-pella.md

```diff
@@ -0,0 +1,8 @@
+# Pella, IA
+
+### Places to eat
+- CUIFS
+- Fred's
```

Comment:
It's the restaurant in the student center at our college. My favorite thing is the buffalo chicken wrap.